opal/mca/hwloc/hwloc191/Makefile.am

```diff
@@ -8,6 +8,13 @@
 # $HEADER$
 #
 
+# Due to what might be a bug in Automake, we need to remove stamp-h?
+# files manually.  See
+# http://debbugs.gnu.org/cgi/bugreport.cgi?bug=19418.
+DISTCLEANFILES = \
+    hwloc/include/hwloc/autogen/stamp-h? \
+    hwloc/include/private/autogen/stamp-h?
```

Comment:
Will the `?` wildcard really work here?  That's surprising to me, since automake doesn't usually permit wildcards in other areas...